src/components/timecard/edit-shift-timecard/EditShiftTimecard.jsx

```diff
@@ -1,18 +1,18 @@
+import { PropTypes } from 'prop-types';
 import { useState } from 'react';
 import { Link } from 'react-router-dom';
-import { useTimecardContext } from '../../../context/TimecardContext';
 import EditShiftHours from '../edit-shift-hours/EditShiftHours';
 
-const EditShiftTimecard = () => {
+const EditShiftTimecard = ({ timeEntry, index }) => {
   const toggleEditShiftHours = (event) => {
     event.preventDefault();
     setShowEditShiftHours(!showEditShiftHours);
   };
 
-  const { timecardData } = useTimecardContext();
-  const timecardDataExists = timecardData.startTime !== '';
+  const timeEntriesExists =
```

Comment:
Should this be named to reflect that a start time exists as it looks like it checks that there is an actual startTime for the given single timeEntry. Rather than mulitple time entries existing, which is what the name suggests.

Reply:
I've changed the this variable to a more suitable name, it's looking for an empty start date, do determine whether the timecard should show an entry in the edit view